app/uk/gov/hmrc/transitmovements/controllers/ArrivalsController.scala

```diff
@@ -76,4 +77,17 @@ class ArrivalsController @Inject() (
       }.toResult
   }
 
+  def getArrivalsForEori(eoriNumber: EORINumber): Action[AnyContent] = Action.async {
+    repo
+      .getMovements(eoriNumber, MovementType.Arrival)
```

Comment:
@drnaylor-oc I think we should call this repo call via service call?  I never seen calling directly repo in controller. That is kind of tight binding between the controller and repository layer. This is just my observer. not sure if I am right were

Reply:
We're doing it elsewhere in this repo -- and it kind of makes sense because this service is _all_ about the Mongo DB and we don't do any additional processing beyond "turn it into a model and send it back in the response". 

Putting a service layer in the middle is a choice we could have made, and we looked at that early on -- but it would have added nothing for a higher level of complexity. So, for this service, I think it's okay.